src/lib/draw-and-pick.js

```diff
@@ -350,25 +350,21 @@ function getPickedColors(gl, {
   pickingFBO,
   deviceRect: {x, y, width, height}
 }) {
-  // TODO - just return glContextWithState once luma updates
   // Make sure we clear scissor test and fbo bindings in case of exceptions
   // We are only interested in one pixel, no need to render anything else
   // Note that the callback here is called synchronously.
-  return glContextWithState(gl, {
+  return withParameters(gl, {
     frameBuffer: pickingFBO,
-    framebuffer: pickingFBO,
-    scissorTest: {x, y, w: width, h: height}
+    scissorTest: true,
+    scissor: [x, y, width, height],
+    blend: true,
+    blendFunc: [gl.ONE, gl.ZERO, gl.CONSTANT_ALPHA, gl.ZERO],
+    blendEquation: gl.FUNC_ADD
+    // TODO - Set clear color
   }, () => {
 
     // Clear the frame buffer
     gl.clear(GL.COLOR_BUFFER_BIT | GL.DEPTH_BUFFER_BIT);
```

Comment:
> Could we eventually move this gl call to luma.gl through withParamters function as well?

Yes makes sense. Although one could argue that clearing the framebuffer is an "action" (i.e not just changing a setting), so it might be better off as a separate luma function? Maybe something @1chandu can look at?